Engine/src/Engine/Scene.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include "../../../thirdparty/entt/entt.h"
+#include "Components.h"
+
+class Entity;
+
+class Scene {
```

Comment:
new line for the bracket.
Also missing namespace

Reply:
done